graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/JpaPredicateBuilder.java

```diff
@@ -262,6 +265,19 @@ protected Predicate getDatePredicate(Path<? extends Date> root, PredicateFilter
         return null;
     }
 
+    private Predicate getUuidPredicate(Path<? extends UUID> field, PredicateFilter filter) {
```

Comment:
You could get rid of <? extends UUID> and use just <UUID> because UUID is a final class and could not have inherited classes.